app/controllers/settings/integration.js

```diff
@@ -61,7 +66,20 @@ export default Controller.extend({
             this.integration.rollbackAttributes();
 
             return transition.retry();
+        },
+
+        deleteIntegration() {
+            this.integration.destroyRecord();
+        },
+
+        confirmIntegrationDeletion(integration) {
+            this.set('integrationToDelete', integration);
```

Comment:
Similar pattern to webhooks, feels unnecessary as there's only ever one integration on a page. But it's useful to show the modal, maybe have a `showIntegrationDeleteModal` instead and set to true/false...?

Reply:
Yeah, a true/false toggle would be clearer here I think